### PR TITLE
Add "female" and other keywords where appropriate

### PR DESCRIPTION
**Pull request by allolex**

### CODE REVIEW

File: emojis.json [hunk 1]
```diff
@@ -131,13 +131,13 @@
...
   "two_men_holding_hands": ["couple", "love", "like", "bromance", "friends"],
-  "two_women_holding_hands": ["friends", "couple", "love", "like"],
+  "two_women_holding_hands": ["friends", "couple", "love", "like", "female"],
   "couplekiss": ["valentines", "love", "like", "dating", "marriage"],
   "couple_with_heart": ["love", "like", "affection", "human", "dating", "valentines", "marriage"],
   "dancers": ["female", "bunny", "women", "girls"],
   "ok_woman": ["women", "girl", "female", "pink", "human"],
   "no_good": ["female", "girl", "woman"],
-  "information_desk_person": ["female", "girl", "woman", "human"],
+  "information_desk_person": ["female", "girl", "woman", "human", "info"],
```
Comment (muan) [inline emojis.json]: This isn't necessary since **info** is already in the name **information_desk_person**.

**Comment by muan** [comment]
Hai thank you! Commented on **info** but all else is good!

**Comment by allolex** [comment]
OK, fixed up the commits to remove the "info" keyword. :smile:

**Comment by muan** [comment]
:sparkles: thank you! :smile:

**Pull request merged**